gemini/tests/set1/datepicker.js

```diff
@@ -106,4 +107,18 @@ gemini.suite('datepicker', child => {
       .setCaptureElements('.clr-example-2')
       .capture('default');
   });
+
+  gemini.suite('datepicker nested in table', child => {
+    child
+      .setUrl('/datepicker/css-regression')
+      .before((actions, find) => {
+        actions.waitForElementToShow('.clr-example-3', WAIT_TIME);
+        actions.wait(WAIT_LOAD_TIME);
+        this.trigger = find('.clr-example-3 .datepicker-trigger');
```

Comment:
Just curious, can anyone tell me why are we using this.trigger instead of local variable here and elsewhere in these tests? Better garbage collection?